### PR TITLE
Categorizes temporal functions

### PR DESCRIPTION
**Pull request by hvub**
This PR categorizes temporal functions. It also factors in all backgrounds. Otherwise, no scenarios have been changed.

Note that Temporal4 contains scenarios that test storage capabilities for temporal values. These could also be categorized under the CREATE clause. However, given that temporal may be considered as advanced functionality, it seems sensible to bundle all its aspects in one category.

### CODE REVIEW

File: tck/features/expressions/temporal/Temporal3.feature [hunk 1]
```diff
@@ -27,12 +27,10 @@
 #
 
 #encoding: utf-8
-Feature: TemporalSelectAcceptance
+Feature: Temporal3 - Create Temporal Values from other Temporal Values
```
Comment (thobe) [inline tck/features/expressions/temporal/Temporal3.feature:30]: I would call this "_Selecting_ Temporal Values from other Temporal Values"

File: tck/features/expressions/temporal/Temporal5.feature [hunk 2]
```diff
@@ -28,12 +28,10 @@
 
 #encoding: utf-8
 
-Feature: TemporalAccessorAcceptance
+Feature: Temporal5 - Access Parts of Temporal Values
```
Comment (thobe) [inline tck/features/expressions/temporal/Temporal5.feature:31]: I think we have called these _Parts_ "_Components_": https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2015-08-06-date-time.adoc#accessing-components-of-temporal-values

So I'd call this feature "Accessing Components of Temporal Values".

Comment (hvub) [reply tck/features/expressions/temporal/Temporal5.feature:31]:
```suggestion
Feature: Temporal5 - Access Components of Temporal Values
```

Yes, great suggestion. Will adopt. Thx.

File: tck/features/expressions/temporal/Temporal3.feature (hunk 1 above)
Comment (hvub) [reply tck/features/expressions/temporal/Temporal3.feature:30]: I thought about that. The original file also the term _selection_. Coming from the natural meaning of the word [_select_](https://www.merriam-webster.com/dictionary/select), the operation portraited in these scenarios allows to do more than just choosing components* to get a new temporal value. It allow to do overwrite components, too. If you wanted to refer to the relational projection, which I can see this operation has similarity too, I would not detour via SQL's SELECT, but go straight to it and speak of `Projecting Temporal Values from other Temporal Values`.

[*] As a side note, while I skimmed these scenarios, I wondered if `datetime`, `date`, and `time` are the only components one can select or if you can also do, say `{hour: other, minute: other}`. Probably not, there are no scenarios for that and `hour`, etc. are expected to be integers, I guess.

Comment (hvub) [reply tck/features/expressions/temporal/Temporal3.feature:30]:
```suggestion
Feature: Temporal3 - Project Temporal Values from other Temporal Values
```

Comment (hvub) [reply tck/features/expressions/temporal/Temporal3.feature:30]: Just saw in [the CIP]( https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2015-08-06-date-time.adoc#247-converting-and-composing-temporal-instant-values) and tried in the product that you can select `hour`, etc. So it is a gap in the scenarios. I will create a card about that.

Comment (hvub) [reply tck/features/expressions/temporal/Temporal3.feature:30]: _Compose_ is another verb that is can be used here and is also used by the [CIP](https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2015-08-06-date-time.adoc#247-converting-and-composing-temporal-instant-values).

```suggestion
Feature: Temporal3 - Compose Temporal Values from other Temporal Values
```

**Pull request merged**